tests/cypress/tests-e2e/layers.cy.js

```diff
@@ -754,6 +754,20 @@ describe('Test of layer handling', () => {
                     const layer = visibleLayers.find((layer) => layer.id === layerId)
                     expect(layer.opacity).to.eq(initialOpacity - step * repetitions)
                 })
+
+                cy.log(
+                    'Check that, for both WMS and WMTS, sliding all the way left gets us an opacity of 0'
+                )
+                visibleLayerIds.forEach((layerId, index) => {
+                    cy.openLayerSettings(layerId)
+                    cy.get(`[data-cy="slider-opacity-layer-${layerId}-${index}"]`)
+                        .should('be.visible')
+                        .realClick({ position: 'left' })
+                    cy.readStoreValue('getters.visibleLayers').should((visibleLayers) => {
+                        const layer = visibleLayers.find((layer) => layer.id === layerId)
+                        expect(layer.opacity).to.eq(0.0)
+                    })
+                })
```

Comment:
Although this test is good if I understand the issue correctly it still don't test the issue, the layers here are simple WMS and WMTS, the issue describe in ticket talk about group of layer, so it need to be tested in the external layers where you have group of WMS layers (geoadmin don't have group of layer in my connaissance)